### DISCUSSION TITLE
[Feature Proposal]: Text shadow utilities

### DISCUSSION THREAD

**Discussion by d8vjork** [tailwindlabs/tailwindcss#307] [Ideas]
Hi there,

I think on a utility that add text shadow just like box shadow, it will be useful maybe if we want to write some text above a block with a background image.

**Comment by benface** [comment]
This can be done with my [`tailwindcss-typography` plugin](https://github.com/benface/tailwindcss-typography), but I believe it should be part of the core. @adamwathan Would you be interested in a PR?

**Comment by steveoh** [comment]
whats the status of this?

**Comment by roydebangshu** [comment]
@steveoh you can use [tailwindcss-textShadow](https://github.com/iunteq/tailwindcss-textShadow)  package.
